Plutarch/Builtin.hs

```diff
@@ -97,4 +97,4 @@ instance PIsData PByteString where
 
 instance PIsData (PBuiltinPair PInteger (PBuiltinList PData)) where
   pfromData x = pasConstr # pforgetData x
-  pdata x = punsafeBuiltin PLC.ConstrData # x
+  pdata x = punsafeBuiltin PLC.ConstrData # (pfstBuiltin # x) #$ psndBuiltin # x
```

Comment:
`plet`

Reply:
Ah, of course - completely forgot `pdata` is a Haskell level function! Done.

Reply:
This is another good reason to recommend Plutarch-level functions: Less accidental duplication.